Makefile

```diff
@@ -545,20 +573,21 @@ help:
 	@echo "   rootfs         builds default EVE rootfs image (upload it to the cloud as BaseImage)"
 	@echo "   rootfs-XXX     builds a particular kind of EVE rootfs image (xen, kvm)"
 	@echo "   live           builds a full disk image of EVE which can be function as a virtual device"
-	@echo "   live-XXX       builds a particular kind of EVE live image (raw, qcow2, gcp)"
+	@echo "   live-XXX       builds a particular kind of EVE live image (raw, qcow2, gcp, parallels)"
 	@echo "   installer      builds raw disk installer image (to be installed on bootable media)"
 	@echo "   installer-iso  builds an ISO installers image (to be installed on bootable media)"
 	@echo
 	@echo "Commonly used run targets (note they don't automatically rebuild images they run):"
-	@echo "   run-compose       runs all EVE microservices via docker-compose deployment"
-	@echo "   run-build-vm      runs a build VM image"
-	@echo "   run-live          runs a full fledged virtual device on qemu (as close as it gets to actual h/w)"
-	@echo "   run-live-gcp      runs a full fledged virtual device on Google Compute Platform (provide your account details)"
-	@echo "   run-rootfs        runs a rootfs.img (limited usefulness e.g. quick test before cloud upload)"
-	@echo "   run-grub          runs our copy of GRUB bootloader and nothing else (very limited usefulness)"
-	@echo "   run-installer-iso runs installer.iso (via qemu) and 'installs' EVE into (initially blank) target.img"
-	@echo "   run-installer-raw runs installer.raw (via qemu) and 'installs' EVE into (initially blank) target.img"
-	@echo "   run-target        runs a full fledged virtual device on qemu from target.img (similar to run-live)"
+	@echo "   run-compose        runs all EVE microservices via docker-compose deployment"
```

Comment:
Please avoid random changes -- make sure only the change you want is reflected (run-live-parallels)

Reply:
I added a line with "run-live-parallels" it's longer than other commands, so moved all "descriptions"  to keep the same line. Of course, I can name it as 'run-live-prl' as  parallels tools "prlctl" and "prlsrvctl"

Reply:
Ah! Didn't realize it -- makes sense then.